aspnetcore/grpc/troubleshoot/sample/8.0/GrpcGreeterClient/Program.cs

```diff
@@ -0,0 +1,203 @@
+﻿#define StandardHTTPS  // Options: StandardHTTPS | IgnoreInvalidCertificate | IgnoreInvalidCertificateClientFactory | CallInsecureGrpcServices | DotNet3InsecureGrpcServices | SubdirectoryHandler
+
+#if StandardHTTPS
+using System.Threading.Tasks;
+using Grpc.Net.Client;
+using GrpcGreeterClient;
+
+// The port number must match the port of the gRPC server.
+#region StandardHTTPS
```

Comment:
Never used regions for snippets, that's banned in this repo.
Snippet names should start with snippet.
cc @tdykstra @guardrex

Reply:
Thanks! I will look through other topics I own to see if I need to correct there as well.  

I was foggy on how we do it for a moment when i did these, so I looked one up on this repo and followed it.  Unfortunately the one I randomly picked used regions, no snipppet in the name.  

`[!code-csharp[](~/fundamentals/minimal-apis/bindingArrays/7.0-samples/todo/Program.cs?name=snippet_batch)]`

If I would have looked around more I would have seen nearly all are done the correct way you mention here.

Reply:
Proper example for me or anyone else can follow: https://github.com/dotnet/AspNetCore.Docs/commit/24e619e0f93d3f810dfc77387c89236be09bb11c

// <snippet_example>

// </snippet_example>